parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -418,6 +418,135 @@ write_files:
     iptables -I FORWARD -d 168.63.129.16 -p tcp --dport 80 -j DROP
     #EOF
 
+- path: /etc/sysctl.d/999-sysctl-aks.conf
+  permissions: "0644"
+  owner: root
+  content: |
+    # This is a partial workaround to this upstream Kubernetes issue: */}}
+    # https://github.com/kubernetes/kubernetes/issues/41916#issuecomment-312428731 */}}
+    net.ipv4.tcp_retries2=8
+    net.core.message_burst=80
+    net.core.message_cost=40
+{{- if ShouldConfigSysctl}}
+{{- if .CustomLinuxOSConfig.Sysctls.NetCoreSomaxconn}}
```

Comment:
can we combine them into one line and save the extra else?

Reply:
I tried {{- if and ShouldConfigSysctl .CustomLinuxOSConfig.Sysctls.NetCoreSomaxconn}}, it will panic at nil pointer